### PR TITLE
fix: replace ReadableStream with Readable

### LINKED ISSUE #240: can't be used inside ts project
```
node_modules/apify-client/dist/resource_clients/key_value_store.d.ts:92:126 - error TS2304: Cannot find name 'ReadableStream'.

92 export declare type ReturnTypeFromOptions<Options extends KeyValueClientGetRecordOptions> = Options['stream'] extends true ? ReadableStream : Options['buffer'] extends true ? Buffer : JsonValue;
                                                                                                                                ~~~~~~~~~~~~~~

node_modules/apify-client/dist/resource_clients/log.d.ts:16:23 - error TS2304: Cannot find name 'ReadableStream'.

16     stream(): Promise<ReadableStream | undefined>;
                         ~~~~~~~~~~~~~~

node_modules/apify-client/dist/utils.d.ts:37:60 - error TS2304: Cannot find name 'ReadableStream'.

37 export declare function isStream(value: unknown): value is ReadableStream;
                                                              ~~~~~~~~~~~~~~
```

```
Found 3 errors in 3 files.

Errors  Files
     1  node_modules/apify-client/dist/resource_clients/key_value_store.d.ts:92
     1  node_modules/apify-client/dist/resource_clients/log.d.ts:16
     1  node_modules/apify-client/dist/utils.d.ts:37
```

the proper shim needs to be added to be able to create the isomorphic module. resorting to `skipLibCheck` is possible but not an actual fix

### PR DESCRIPTION
**Pull request by foxt451**
The first one is from DOM, the latter - Node. Also, added type for URLSearchParams arg as it's different in Node and won't accept string[][].

Tested by running build script and commenting out DOM in tsconfig libs

Closes #240

### CODE REVIEW

**Review by B4nan — APPROVED** [review]

File: src/resource_clients/key_value_store.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import log from '@apify/log';
 import ow from 'ow';
 import { JsonValue } from 'type-fest';
+import { Readable } from 'stream';
```
[the remark this answers is not in the document]
Comment (B4nan) [reply src/resource_clients/key_value_store.ts:4]: hmm now i think of this, dont we need to configure the webpack build too to support `stream`? or is that client side bundle not working at all? :D 

https://github.com/apify/apify-client-js/blob/master/webpack.config.js#L51

[the remark this answers is not in the document]
Comment (foxt451) [reply src/resource_clients/key_value_store.ts]: Well, I've checked other places and node modules already were imported, like 'os'

[the remark this answers is not in the document]
Comment (B4nan) [reply src/resource_clients/key_value_store.ts]: i think (older versions of) webpack adds some polyfills automatically, but not really sure.

(and we use older version here)

but maybe it was broken for a long time already and nobody uses it

maybe we could do this? https://stackoverflow.com/a/65556946/3665878

[the remark this answers is not in the document]
Comment (B4nan) [reply src/resource_clients/key_value_store.ts]: ok sounds good!

**Pull request merged**